vault/iammfa.go

```diff
@@ -0,0 +1,198 @@
+package vault
```

Comment:
this no longer has anything to do with "vault"ing, so might fit better in a different package?

Reply:
will change